### DISCUSSION TITLE
在其他网站发现了这个软件，希望作者确认一下是不是被盗传的

### DISCUSSION THREAD

**Discussion by MeAquAlex** [xiaoyaocz/flutter_dmzj#236] [General]
在谷歌搜索再漫画的时候这玩意蹦到了首页，感觉不像是作者主动上传的。

网址：http://m.xlhs.com/app/111751.html
![Screenshot_20240923-122547](https://github.com/user-attachments/assets/30b628da-eff4-4c98-9642-975110645816)
![Screenshot_20240923-122135](https://github.com/user-attachments/assets/e1e73353-08b5-4aa1-b243-e714d0b4e433)
![Screenshot_20240923-122145](https://github.com/user-attachments/assets/15a6104b-3dfd-4fac-a05d-1bad48c69a08)

**Comment by xiaoyaocz** [comment]
只在Github上发布
